src/cli.rs

```diff
@@ -45,6 +45,10 @@ pub enum Subcommands {
         /// Show full MQTT communication
         #[arg(short, long)]
         verbose: bool,
+
+        /// Output logs as newline-delimited JSON
+        #[arg(short, long)]
+        json: bool,
```

Comment:
I would put json before verbose as its alphabetically before (also in other places where verbose and json are passed). topic is not a `--flag` so I would keep it seperate.